### PR TITLE
[Wrangler] Show deployments command

### PR DESCRIPTION
**Pull request by lrapoport-cf**

### CODE REVIEW

**Review by caass — APPROVED** [review]
ok

File: content/workers/wrangler/commands.md [hunk 1]
```diff
@@ -1199,9 +1200,6 @@ $ wrangler deployments
 
 {{</definitions>}}
 
-TODO Add examples of logged output
```
Comment (jspspike) [inline content/workers/wrangler/commands.md:1202]: Do we not want to put example output here? Otherwise looks good

**Review by jspspike — APPROVED** [review]

File: content/workers/wrangler/commands.md [hunk 2]
```diff
@@ -154,9 +154,10 @@ None of the options for this command are required. Many of these options can be
 - `--tsconfig` {{<type>}}string{{</type>}}
   - Path to a custom `tsconfig.json` file.
 - `--local` {{<type>}}boolean{{</type>}} {{<prop-meta>}}(default: false){{</prop-meta>}}
+
   - Run the preview of the Worker directly on your local machine.
     {{<Aside type="warning">}}
-This runs an ephemeral local version of your Worker, and will not be able to access data stored on Cloudflare's Edge (for instance, this includes your data stored on KV). If you'd like to persist data locally, using the `--persist` flag will tell Wrangler to store data in the `.wrangler/state` subdirectory.
+    This runs an ephemeral local version of your Worker, and will not be able to access data stored on Cloudflare's Edge (for instance, this includes your data stored on KV). If you'd like to persist data locally, using the `--persist` flag will tell Wrangler to store data in the `.wrangler/state` subdirectory.
```
Comment (rozenmd) [inline content/workers/wrangler/commands.md]:
```suggestion
This runs an ephemeral local version of your Worker, and will not be able to access data stored on Cloudflare's Edge (for instance, this includes your data stored on KV). If you'd like to persist data locally, using the `--persist` flag will tell Wrangler to store data in the `.wrangler/state` subdirectory.
```

**Review by deadlypants1973 — APPROVED** [review]

File: content/workers/wrangler/commands.md (hunk 1 above)
Comment (lrapoport-cf) [reply content/workers/wrangler/commands.md:1202]: @jspspike that's a good follow up improvement -- will get this merged in now so it's documented and then create a ticket to add example output 👍

**Comment by lrapoport-cf** [comment]
this PR no longer necessary as changes have been addressed in https://github.com/cloudflare/cloudflare-docs/pull/6631

**Pull request closed without merging**